src/nvim/edit.c

```diff
@@ -609,9 +609,6 @@ static int insert_execute(VimState *state, int key)
   InsertState *s = (InsertState *)state;
   s->c = key;
 
-  // Don't want K_EVENT with cursorhold for the second key, e.g., after CTRL-V.
-  did_cursorhold = true;
```

Comment:
I think the reason this line exists is because the old `K_CURSORHOLD` key is not handled while processing commands like `<c-v>`, and neither is `K_EVENT`. For example, what happens if you enter insert mode, type ctrl-v and then go into a separate python console and send a msgpack-rpc request to nvim?

Reply:
Refactoring / fixing this properly seems reasonable.
But a minimal patch like https://github.com/neovim/neovim/issues/3757#issuecomment-497279946 could maybe do for now - after all `os_inchar` times out based on `&updatetime` (used for CursorHold(I)), and therefore might be the best place to trigger/set it still.